Formula/collector-sidecar.rb

```diff
@@ -0,0 +1,31 @@
+class CollectorSidecar < Formula
+  desc "Manage log collectors through Graylog"
+  homepage "https://github.com/Graylog2/collector-sidecar"
+  url "https://github.com/Graylog2/collector-sidecar/archive/0.0.9.tar.gz"
+  sha256 "a854d84bf65e3fb9409743869701dab5c059d14d0b615acd14c8923bef30bc9f"
+
+  depends_on "glide" => :build
+  depends_on "go" => :build
+  depends_on :hg => :build
```

Comment:
That was what I tried first, but brew audit gave me an error...
I tried:
```
  depends_on "hg" => :build
```
brew audit said:
```
  * Dependency 'hg' is an alias; use the canonical name 'mercurial'.
```
I then tried:
```
  depends_on "mercurial" => :build
```
brew audit said:
```
  * Use `depends_on :hg` instead of `depends_on 'mercurial'`
```
I can make the change, but the formula won't pass audit, what's the preferred approach?